docs/recipes/keypairs-and-wallets.md

```diff
@@ -302,4 +302,4 @@ To do so we will import the [TweetNaCl][1] crypto library.
 
 </SolanaCodeGroup>
 
-[1]: https://www.npmjs.com/package/tweetnacl
\ No newline at end of file
+[1]: https://www.npmjs.com/package/tweetnacl
```

Comment:
Damnit. I didn't touch this line. I just used the ‘edit on GitHub’ feature and this happened.